images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,20 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json

```

Comment:
@AlenaSviridenko @maxim-lobanov does it make sense to add some warning that this script should be executed only after `Install-WDK` since one of the extensions is being downloaded as a part of WDK?